src/spatiotemporalpaths/CSVPiecewiseLinearSpatioTemporalPath.C

```diff
@@ -0,0 +1,63 @@
+/****************************************************************************/
+/*                        DO NOT MODIFY THIS HEADER                         */
+/*                                                                          */
+/* MALAMUTE: MOOSE Application Library for Advanced Manufacturing UTilitiEs */
+/*                                                                          */
+/*           Copyright 2021 - 2023, Battelle Energy Alliance, LLC           */
+/*                           ALL RIGHTS RESERVED                            */
+/****************************************************************************/
+
+#include "CSVPiecewiseLinearSpatioTemporalPath.h"
+#include "DelimitedFileReader.h"
+
+registerMooseObject("MalamuteApp", CSVPiecewiseLinearSpatioTemporalPath);
+
+InputParameters
+CSVPiecewiseLinearSpatioTemporalPath::validParams()
+{
+  InputParameters params = PiecewiseLinearSpatioTemporalPathBase::validParams();
+  params.addClassDescription("A piecewise linear spatiotemporal path from a csv file.");
+  params.addRequiredParam<FileName>(
+      "file", "CSV file containing information about the spatio-temporal path");
+  MooseEnum header_flag("OFF ON AUTO", "AUTO");
+  params.addParam<MooseEnum>("file_header",
+                             header_flag,
+                             "Set the header flag. ON: use the first row has header, OFF: assumes "
+                             "no header, AUTO: attempt to determine if a header exists.");
+  MooseEnum format_flag("COLUMNS ROWS", "COLUMNS");
+  params.addParam<MooseEnum>("file_format", format_flag, "Set the file format (rows vs. columns).");
```

Comment:
We should let the user know the default enum case for both of these params. I don't remember seeing such a mention in the documentation, so it should be done either here in the param description or in the markdown.